app/services/calculate_candidate.rb

```diff
@@ -0,0 +1,14 @@
+class CalculateCandidate
+  def initialize(answers)
+    @answers = answers
+  end
+
+  def execute
+    answers_by_candidates = @answers.group_by(&:candidate)
+    max = (answers_by_candidates.map { |k, v| v.count }).max
+    (answers_by_candidates.map { |k, v| (v.count == max) ? 
+                                 k :
+                                 0 
+                                }).select { |a| a != 0 }
```

Comment:
`end` at 12, 32 is not aligned with `answers_by_candidates.map { |k, v| (v.count == max) ? ` at 9, 5 or `(answers_by_candidates.map { |k, v| (v.count == max) ? ` at 9, 4